client/modules/sheaf-core/src/extensions/spellcheck/spellchecker/normalize.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Pairs of strings, with the first string being text to replace with the
+ * latter. Used for normalizing text.
+ */
+const REPLACEMENTS: [string, string][] = [
+  ["’", "'"] // normalize right single quotation marks into apostrophes
+]
+
+/** Returns a list of patterns to filter out as "not words", depending on locale. */
+function getFilters(locale: string): RegExp[] {
+  switch (locale) {
+    case "en":
```

Comment:
Does this capture `en_GB` etc.?

Reply:
It's intended to only handle the language code. You're supposed to feed into the spellchecker just a locale's language, so by the time it gets to this function it shouldn't have any region codes attached.

That could cause problems later, but honestly, I don't really know. It adds quite a bit of complexity if regions are supported, because then we start talking about having to merge dictionaries (probably) or worse.